.gitignore

```diff
@@ -6,4 +6,7 @@ _build/
 _sbuild/
 *.o
 *.a
-/z3_problems
+z3_problems
+
+# Typical CMake build directory.
+/build
```

Comment:
Could you also add `/cmake-build-*` which is the default for CLion (it creates cmake-build-debug/cmake-build-release, etc. depending on the CMake mode you choose.

Reply:
Added